### PR TITLE
modify some codes of admission controller

### PR DESCRIPTION
**Pull request by gy95**
Signed-off-by: gy95 <guoyao17@huawei.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by fisherxu — COMMENTED** [review]
/approve

File: cloud/pkg/admissioncontroller/admit_ruleendpoint.go [hunk 1]
```diff
@@ -3,47 +3,41 @@ package admissioncontroller
...
+			return toAdmissionResponse(err)
 		}
 		err := validateRuleEndpoint(&ruleEndpoint)
 		if err != nil {
-			msg = err.Error()
-			break
+			return toAdmissionResponse(err)
 		}
 		reviewResponse.Allowed = true
 		klog.Info("admission validation passed!")
```
Comment (zc2638) [inline cloud/pkg/admissioncontroller/admit_ruleendpoint.go:29]: Is this `info log` necessary?
If it is not useful, can you consider deleting it?

File: cloud/pkg/admissioncontroller/admit_ruleendpoint.go [hunk 2]
```diff
@@ -3,47 +3,41 @@ package admissioncontroller
...
+		return &reviewResponse
 	case admissionv1beta1.Delete, admissionv1beta1.Connect:
 		//no rule defined for above operations, greenlight for all of above.
 		reviewResponse.Allowed = true
 		klog.Info("admission validation passed!")
```
Comment (zc2638) [inline cloud/pkg/admissioncontroller/admit_ruleendpoint.go:34]: too

File: cloud/pkg/admissioncontroller/admit_ruleendpoint.go (hunk 1 above)
Comment (gy95) [reply cloud/pkg/admissioncontroller/admit_ruleendpoint.go]: fixed.

**Pull request merged**